includes/Admin/Sync_Page.php

```diff
@@ -376,7 +418,7 @@ private static function output_sync_with_square_modal_template() {
 							<?php
 							printf(
 								/* translators: Placeholders: %1$s - the name of the system of record set in the sync settings (e.g. Square or WooCommerce), %3%s - unordered HTML list of additional information item(s) */
-								esc_html__( 'You are about to sync products with Square. %1$s is your system of record set in the sync settings. For all products synced with Square: %2$s', 'woocommerce-square' ),
+								esc_html__( 'You are about to sync existing products with Square. %1$s is your system of record set in the sync settings. For all products currently synced with Square: %2$s', 'woocommerce-square' ),
```

Comment:
```suggestion
								esc_html__( 'You are about to sync products with Square. %1$s is your system of record set in the sync settings. For all products synced with Square: %2$s', 'woocommerce-square' ),
```

As we are adding notice only, better to keep this as it is or only update if Square is SOR.